st2common/st2common/models/api/execution.py

```diff
@@ -121,6 +137,9 @@ def from_model(cls, model, mask_secrets=False):
             end_timestamp = isotime.format(end_timestamp, offset=False)
             doc['end_timestamp'] = end_timestamp
 
+        for entry in doc['log']:
```

Comment:
Will this work since we don't default it to an empty array? Maybe safer to do `entry.get('log', []) or []` or something like that.